lesson8-UnitTesting/FizzBuzz/FizzBuzzConsole/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Linq;
+
+namespace FizzBuzzConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            static string Generate(int number)
```

Comment:
Why not use `FizzBuzz.Generate` instead?

Reply:
done